.travis.yml

```diff
@@ -17,15 +19,15 @@ matrix:
   allow_failures:
     - php: 7.0
 
-install:
-  - sudo apt-get update
-  - sudo apt-get -qq --fix-missing install valgrind
-
-  - wget -qO - http://www.rabbitmq.com/rabbitmq-signing-key-public.asc | sudo apt-key add -
-  - sudo add-apt-repository 'deb http://www.rabbitmq.com/debian/ testing main'
-  - sudo apt-get update
-  - sudo apt-get install --only-upgrade -y rabbitmq-server
-  - sudo service rabbitmq-server restart
+#install:
+#  - sudo apt-get update
+#  - sudo apt-get -qq --fix-missing install valgrind
+#
+#  - wget -qO - http://www.rabbitmq.com/rabbitmq-signing-key-public.asc | sudo apt-key add -
+#  - sudo add-apt-repository 'deb http://www.rabbitmq.com/debian/ testing main'
+#  - sudo apt-get update
+#  - sudo apt-get install --only-upgrade -y rabbitmq-server
+#  - sudo service rabbitmq-server restart
```

Comment:
this should be removed rather than being commented. The history is already kept by git, and it is much better at it than comments